src/styles/fonts.css

```diff
@@ -0,0 +1,3 @@
+/* Imports Work Sans text */
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans&display=swap');
```

Comment:
All of the font types were directly downloaded to be compatible with more browsers! So we don't need to import them from google